src/core/qgsabstractdatabaseproviderconnection.h

```diff
@@ -494,6 +496,16 @@ class CORE_EXPORT QgsAbstractDatabaseProviderConnection : public QgsAbstractProv
      */
     virtual QStringList schemas() const SIP_THROW( QgsProviderConnectionException );
 
+    /**
+     * Returns the fields of a \a table and \a schema.
+     * Raises a QgsProviderConnectionException if any errors are encountered.
+     * \note the default implementation creates a temporary vector layer, providers may
+     * choose to override this method for a greater efficiency.
+     * \since QGIS 3.18
```

Comment:
```suggestion
     * \since QGIS 3.16
```
Do not wait so long ;-)

Reply:
yeah, this is actually the first pisce of a more ambitious project but I guess we can merge it soon.